packages/build/custom_typings/workbox.d.ts

```diff
@@ -0,0 +1,69 @@
+interface ManifestEntry {
+  url: string;
+  revision: string;
+}
+
+declare module "workbox-build" {
```

Comment:
Since you all own Workbox, would it be better to move the typings over there?

Reply:
@jeffposnick Thoughts?

Reply:
@jeffposnick I guess we can go ahead with keeping it here and update if we ever move this inside workbox

Reply:
Looping back to this: the Workbox team's stance on `.d.ts` is reflected at https://github.com/GoogleChrome/workbox/pull/1436#issuecomment-396678604

None of the core maintainers are familiar with TypeScript today, and we don't feel comfortable with the guarantees of support that we'd signal if we hosted the definitions.

Either checking things into this repo, or alternatively hosting them on https://definitelytyped.org/, seems like the best course of action at this time.